### ISSUE TITLE
Some paragraph from MS Word turns into ul elements

### ISSUE THREAD

**Issue by thepriefy** [ckeditor/ckeditor4#3627]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Copy text from MS Word

    “Lorem!!”
 
    “ipsum!!!”   dolor sit amet (consectetur)        

![01](https://user-images.githubusercontent.com/10849586/68199523-c7c01900-fff0-11e9-8bed-5aa45240d096.PNG)

2. Paste on CKEditor

### Expected result

I copied 2 lines of text from MS Word, expected 2 paragraph in CKEditor.

### Actual result

One paragraph and one un-ordered list (ul). Some part of text also disappeared.

“Lorem!!”

(ul) “ipsum dolor sit amet (consectetur)

![02](https://user-images.githubusercontent.com/10849586/68200127-d5c26980-fff1-11e9-9cb6-864692b81912.PNG)

## Other details

* Browser: Google Chrome Version 77.0.3865.120 (I tried on Firefox and the result is the same.)
* OS: Windows 10 (64-Bit)
* CKEditor version: 4
* Installed CKEditor plugins: footnotes, horizontalrule, confighelper, videodetector, divarea, colordialog

I logged the result from editor.on('paste', function (e) {...}); function.

**e.data.dataTransfer._.data.text/html:**

![03](https://user-images.githubusercontent.com/10849586/68200024-aa3f7f00-fff1-11e9-9155-1b258895a4f5.PNG)

**e.data.dataValue:**
`"<p style="margin-bottom:.0001pt"><span style="line-height:normal"><span style="font-size:18.0pt"><span style="font-family:&quot;Calibri Light&quot;,sans-serif"><span style="color:black"> “Lorem!!” </span></span></span><span style="font-size:9.0pt"><span style="font-family:&quot;Calibri Light&quot;,sans-serif"><span style="color:black"></span></span></span></span></p><p style="margin-bottom:.0001pt"><span style="line-height:normal"> <span style="font-size:9.0pt"><span style="font-family:&quot;Calibri Light&quot;,sans-serif"><span style="color:black"></span></span></span></span></p><p><ul><li style="margin-bottom:.0001pt"><span style="line-height:normal"><span style="font-size:18.0pt"><span style="font-family:&quot;Calibri Light&quot;,sans-serif"><span style="color:black"> “ipsum dolor sit amet (consectetur) </span></span></span> </span></li></ul></p>"`

**Comment by msamsel** [comment]
Hi,

I'm not able to reproduce the bug. I used this sample page for testing purpose: https://ckeditor.com/latest/samples/index.html

I noticed that you're using 3rd party plugin not developed by CKEditor team. There is a probability that one of those plugins introduces this wrong behaviour.
Please check if the issue occurs when there is no 3rd party plugins in your setup. If that happens please provide more details about your case. You could attach the document which you try to copy-paste and also provide some simple demo page (it can be on codepen.io), with your editor setup.

**Comment by thepriefy** [comment]
@msamsel Than you for your reply. I have tested on the sample page and it looks normal. Now I removed all plugins in my editor but it still happen. 

This is what my CKEditor looks like when I removed all plugins.

![04](https://user-images.githubusercontent.com/10849586/68378286-6ded5980-017e-11ea-815d-cdffb55701d2.PNG)

Is there any different between version 4.6.2 (mine) and 4.13.0 (as I see in https://ckeditor.com/ckeditor-4/download)?

Now, I'm trying to change my CKEditor to the latest version (v4.13.0). Do I need to replace my current files with files I downloaded from [this page](https://ckeditor.com/ckeditor-4/download/)?
